### PR TITLE
Move GitHub specific files into .github folder (issue #6)

### PR DESCRIPTION
**Pull request by eugener**
- Move `CONTRIBUTING.md` into `.github` folder
- Remove original `README` file
- Create new `README.md` in `.github` folder

### CODE REVIEW

**Comment by bourgesl** [comment]
The new README looks great, thanks

**Comment by eugener** [comment]
@kevinrushforth @johanvos Please review

File: .github/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Open JFX Mirror   
-[![Build Status](https://travis-ci.org/javafxports/openjdk-jfx.svg?branch=develop)](https://travis-ci.org/javafxports/openjdk-jfx) ![Build Status](https://ci.appveyor.com/api/projects/status/javafxports-github-bot/openjdk-jfx/branch/develop?svg=true)
+[![Build Status](https://travis-ci.org/javafxports/openjdk-jfx.svg?branch=develop)](https://travis-ci.org/javafxports/openjdk-jfx) [![Build Status](https://ci.appveyor.com/api/projects/status/javafxports-github-bot/openjdk-jfx/branch/develop?svg=true)](https://ci.appveyor.com/project/javafxports-github-bot/openjdk-jfx)
 
 OpenJFX is an open source, next generation client application platform for desktop and embedded systems based on JavaSE. It is a collaborative effort by many individuals and companies with the goal of producing a modern, efficient, and fully featured toolkit for developing rich client applications. This is the open source project where we develop JavaFX.
```
Comment (johanvos) [inline .github/README.md]: desktop, mobile and embedded?

Comment (eugener) [reply .github/README.md]: Sounds good... I simply took it from OpenJFX Wiki. 
IMO we can always tweak the ream file later. The main point of this PR is structure and build status.

**Comment by eugener** [comment]
Updated...please merge at your convenience.

**Pull request merged**